stubs/phantom/action_result.py

```diff
@@ -0,0 +1,52 @@
+from phantom.app import APP_ERROR
+
+class ActionResult:
```

Comment:
Is this the actual decompiled source for this file? Or a stub you wrote with a mostly-functional implementation that matches the behavior of the actual class?

Reply:
The latter, though I did determine this using decompilers